tests/functional/event_handler/_pydantic/test_openapi_params.py

```diff
@@ -172,6 +172,42 @@ def handler() -> Response[Annotated[str, Body(title="Response title")]]:
     assert response.schema_.type == "string"
 
 
+def test_openapi_with_tuple_returns():
+    app = APIGatewayRestResolver()
+
+    @app.get("/")
+    def handler() -> tuple[str, int]:
+        return "Hello, world", 200
+
+    schema = app.get_openapi_schema()
+    assert len(schema.paths.keys()) == 1
+
+    get = schema.paths["/"].get
+    assert get.parameters is None
+
+    response = get.responses[200].content[JSON_CONTENT_TYPE]
+    assert response.schema_.title == "Return"
+    assert response.schema_.type == "string"
+
+
+def test_openapi_with_tuple_annotated_returns():
+    app = APIGatewayRestResolver()
+
+    @app.get("/")
+    def handler() -> tuple[Annotated[str, Body(title="Response title")], int]:
```

Comment:
Can you pls import `Tuple` from typing? I need to migrate our tests to adopt the new type annotation, but we have some issues for now.

Reply:
Done.